frontend/src/views/account-mgmt/ManageFacilityView.vue

```diff
@@ -140,7 +140,7 @@
       :defaultRequestCategoryId="getRequestCategoryIdByName(REQUEST_CATEGORY_NAMES.ACCOUNT_MAINTENANCE)"
       :defaultFacility="facility"
       @close="toggleChangeRequestDialog"
-      @close-confirmation="handleCloseConfirmation" />
+      @submit-phone-email="handleCRSubmit" />
```

Comment:
here you have the event in kebab case, but the emit is done in camelCase. It still works regardless, but is it better to have them match? Just curious because it seems like there are multiple ways we name emits/events in the project

Reply:
It's best to follow https://vuejs.org/guide/components/events.html which borrows from https://vuejs.org/guide/components/props.html#prop-name-casing. The project is admittedly a mix.